services/skus/receipt.go

```diff
@@ -14,53 +14,24 @@ import (
 	"time"
 
 	"github.com/awa/go-iap/appstore"
-	"github.com/awa/go-iap/playstore"
-	"google.golang.org/api/androidpublisher/v3"
-
 	appctx "github.com/brave-intl/bat-go/libs/context"
 	"github.com/brave-intl/bat-go/libs/logging"
+	"google.golang.org/api/androidpublisher/v3"
```

Comment:
nit/suggestion: Would something like this make import paths slightly shorter and manageable?

```suggestion
	androidpub "google.golang.org/api/androidpublisher/v3"
```